doc/source/whatsnew/v1.1.1.rst

```diff
@@ -49,7 +49,7 @@ Categorical
 
 **Indexing**
 
--
+- Bug in series.truncate when trying to truncate a single-element series (:issue:`35544`)
```

Comment:
should be ``` :meth:`Series.truncate` ```

Reply:
Thanks for the review. This has been updated.